src/components/Search.js

```diff
@@ -0,0 +1,89 @@
+import React, { Fragment, useState, useEffect } from 'react';
+import { Global, css } from '@emotion/core';
+import { DocSearch } from '@docsearch/react';
+
+const Search = () => (
+  <Fragment>
+    <Global
+      styles={css`
+        :root {
+          --docsearch-primary-color: #ff6b81;
+          --docsearch-text-color: rgb(245, 246, 247);
+          --docsearch-container-background: rgba(9, 10, 17, 0.8);
+          --docsearch-modal-background: #281e36;
+          --docsearch-modal-shadow: inset 1px 1px 0 0 rgb(44, 46, 64),
+            0 3px 8px 0 rgb(0, 3, 9);
+          --docsearch-searchbox-background: rgb(9, 10, 17);
+          --docsearch-searchbox-focus-background: #000;
+          --docsearch-hit-color: rgb(190, 195, 201);
+          --docsearch-hit-shadow: none;
+          --docsearch-hit-background: rgb(9, 10, 17);
+          --docsearch-key-gradient: linear-gradient(
```

Comment:
The contrast on the keys is also little.

We can either lighten the SVG content or darken the gradient?

Reply:
Good point. These values are closed to the default values (#969faf / #ebedf0). [Using a contrast checker](https://webaim.org/resources/contrastchecker/) shows that these values should be also edited to have a proper accessibility.
![image](https://user-images.githubusercontent.com/32097720/91152003-aedb3e00-e6be-11ea-9831-4a607d0d96bb.png)

Reply:
I have made contrast consistent among SVG and placeholder via 65f6460